AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/list/participant-list.component.ts

```diff
@@ -26,8 +26,8 @@ export class ParticipantListComponent implements OnInit, OnChanges, DoCheck {
     constructor(private logger: Logger, private videoHearingsService: VideoHearingsService) {}
 
     ngDoCheck(): void {
-        const containsNewParticipants = this.hearing.participants?.some(hearingParticipant => this.sortedParticipants.includes(hearingParticipant)) ?? false;
-        const containsRemovedParticipants = this.sortedParticipants?.some(sortedParticipant => !this.hearing.participants.includes(sortedParticipant)) ?? false;
+        const containsNewParticipants = !this.hearing?.participants?.every(hearingParticipant => this.sortedParticipants.includes(hearingParticipant)) ?? false;
+        const containsRemovedParticipants = !this.sortedParticipants?.every(sortedParticipant => this.hearing.participants.includes(sortedParticipant)) ?? false;
```

Comment:
Would the following not work? I feel it's a bit easier to parse
` const containsNewParticipants = this.hearing?.participants?.some(hearingParticipant => !this.sortedParticipants.includes(hearingParticipant)) ?? false;
        const containsRemovedParticipants = this.sortedParticipants?.some(sortedParticipant => !this.hearing.participants.includes(sortedParticipant)) ?? false;`

Reply:
Matter of preference. Happy to approve as is